src/test/java/uk/gov/companieshouse/officerfiling/api/controller/DirectorsControllerImplIT.java

```diff
@@ -69,8 +76,10 @@ class DirectorsControllerImplIT {
     private OfficerService officerService;
     @Mock
     private HttpServletRequest request;
-    @Mock
     private Transaction transaction;
+
+    @Mock
+    private TransactionService transactionService;
     @Mock
     Optional<OfficerFiling> officerFilingOptional;
```

Comment:
Tech debt:
Optionals not meant to be used for fields.  Better to use explicit stub values (see further comments below).